enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/OnboardingServiceImpl.java

```diff
@@ -458,4 +438,51 @@ private OnboardingProcessEntity resumeExistingProcess(final OnboardingProcessEnt
         }
         return process;
     }
-}
\ No newline at end of file
+
+    private String parseIdentificationData(final Map<String, Object> identification) throws InvalidRequestObjectException {
+        try {
+            return normalizedMapper.writeValueAsString(identification);
+        } catch (JsonProcessingException ex) {
+            logger.warn("Invalid identification data: {}", identification);
+            throw new InvalidRequestObjectException();
+        }
+    }
+
+    private static boolean isFakeUser(final String userId) {
+        return userId.startsWith(FAKE_USER_ID_PREFIX);
```

Comment:
The question is, whether such differentiation of fake user is solid enough. I guess so.